pom.xml

```diff
@@ -167,6 +167,38 @@
 			<optional>true</optional>
 		</dependency>
 
+		<dependency>
```

Comment:
The Kotlin reflect and JDK dependencies support JSON serialization and JDK 8 libraries.

The coroutine support is used to improve the integration to the CF Reactive code.

`kotlin-logging-jvm` is a Kotlin wrapper around SLF4J to make the logging API cleaner.